testdata/networking/multus-cni/NetworkAttachmentDefinitions/IPv6/macvlan-bridge-v6.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: macvlan-bridge-v6
+spec:
+  config: '{
+      "cniVersion": "0.3.0",
+      "type": "macvlan",
+      "master": "ens3",
+      "mode": "bridge",
+      "ipam": {
+        "type": "host-local",
+        "subnet": "fd00::192:168:22:0/112",
+        "rangeStart": "fd00::192:168:22:100",
+        "rangeEnd": "fd00::192:168:22:200"
```

Comment:
these addresses are a little confusing because IPv6 is hex and not the regular 192.168.22.100.

it might make more sense to keep it all hex and use one of the well-know fake hex addresses.

  `fd00::dead:beef:0/112`  or  `fd00::bad:bad:0/112`  or something from https://en.wikipedia.org/wiki/Hexspeak

Reply:
Good idea, update to use fd00:dead:beef::/64 right now.